src/data/set/countable.lean

```diff
@@ -6,7 +6,7 @@ Author: Johannes Hölzl
 Countable sets.
 -/
 
-import data.equiv.list data.set.finite logic.function data.set.function
+import data.equiv.list data.set.finite logic.function data.set.function topology.basic
```

Comment:
This file (rather low in the hierarchy) should not import topology. Instead, let some file in topology import this file, and move the relevant topological lemma there.